src/components/grid-aware/ArticleSpotlightCard/ArticleSpotlightCard.js

```diff
@@ -39,7 +39,7 @@ const ArticleSpotlightCard = ({
     <div
       className={s.bleedBackground}
       style={{
-        background: `center / cover no-repeat url("${backgroundImage}")`,
+        background: `bottom / cover no-repeat url("${backgroundImage}")`,
```

Comment:
I'm not sure if this is a good idea. Won't this interfere with the Article Spotlight on the Volunteer page, which has a different image?

Could we add that component to the Volunteer page first so that we know what the impact of making any further CSS tweaks to the component?